File: week-03/day-3/06.py
# create a function that takes a list and returns a new list that is reversed
def list_taking(list):
    newl = []
    for i in range(len(list)-1, -1, -1):
        newl += [list[i]]
    return newl

my_list = [1, 2, 3]
print(list_taking(my_list))

File: week-07/day-3/06.js
'use strict';


// create a function that takes a string and a letter and returns a boolean
// it should return true if the string consits the given letter, false otherwise

function letterInWord(word, letter) {
  return word.split('').some(function(char) {
    return letter === char;
  });
}

console.log(letterInWord("pear", "a"));
